validator/src/main.rs

```diff
@@ -2013,6 +2013,16 @@ pub fn main() {
                 .takes_value(true)
                 .help("Number of bins to divide the accounts index into"),
         )
+        .arg(
+            Arg::with_name("accounts_index_path")
+                .long("accounts-index-path")
+                .value_name("PATH")
+                .takes_value(true)
+                .multiple(true)
+                .help("Persistent accounts-index location. \
+                       May be specified multiple times. \
+                       [default: [ledger]/accounts_index"),
```

Comment:
Should there be a `]` at the end of the default line? (same for ledger-tool too)